packages/data-provider/test/utils/auth.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
```

Comment:
TODO: switch to Node test runner, to decrease external dependencies
e.g. `node:test` and `node:assert`